eoxserver/resources/coverages/metadata/coverage_formats/eoom.py

```diff
@@ -55,7 +58,8 @@
 nsmap_gml = NameSpaceMap(NS_GML)
 
 
-class EOOMFormatReader(object):
+class EOOMFormatReader(Component):
+    implements(MetadataReaderInterface)
```

Comment:
These lines should not be necessary. The `Interface` and `Implementation` logic was removed since v0.4